bpf/lib/lb.h

```diff
@@ -663,8 +663,9 @@ static __always_inline int lb6_xlate(struct __ctx_buff *ctx,
 	if (skip_l3_xlate)
 		goto l4_xlate;
 
-	ipv6_store_daddr(ctx, new_dst->addr, l3_off);
-	if (csum_off) {
+	if (ipv6_store_daddr(ctx, new_dst->addr, l3_off) < 0)
+		return DROP_WRITE_ERROR;
+	if (csum_off->offset) {
```

Comment:
Can you please drop the change to `csum_off->offset`? If we want this in v1.12, we should explicitly backport https://github.com/cilium/cilium/pull/22953. The assumption so far was that it's not needed (because no SCTP support in v1.12).